README.md

```diff
@@ -68,6 +86,14 @@ ruby -r vernier -e 'Vernier.trace_retained(out: "irb_profile.json") { require "i
 
 Retained-memory flamegraphs must be interpreted a little differently than a typical profiling flamegraph. In a retained-memory flamegraph, the x-axis represents a proportion of memory in bytes,  _not time or samples_ The topmost boxes on the y-axis represent the retained objects, with their stacktrace below; their width represents the percentage of overall retained memory each object occupies.
 
+### Options
+
+Option | Description
+:- | :-
+`mode` | The sampling mode to use. One of `:wall`, `:retained` or `:custom`. Default is `:wall`.
```

Comment:
I'm not really sure how the custom collector is meant to be used, I'm guessing the native implementation is meant to be extended? Let me know if it shouldn't be included here just yet / if there's more information that can be added.

Reply:
I'm guessing nobody has used it, and I'd probably implement it differently today, the intention was to be like stackprof's `:custom` mode where you call `StackProf.sample` at times you want to measure.

Reply:
Ah right. Similar to the hooks, I'll remove the `:custom` option from this list for now and it can be added in once implementation has been finalised.